roles/edpm_libvirt/molecule/default/prepare.yml

```diff
@@ -176,6 +176,7 @@
       community.crypto.openssl_csr_pipe:
         privatekey_path: "{{ edpm_libvirt_tls_cert_src_dir }}/tls.key"
         organization_name: fake-corp
+        common_name: "{{ inventory_hostname }}"
```

Comment:
this shoudl work as i used it in the subject_alt_name previously
to pass the check before they reworte it.

if not we might need to use hostvars or regestert the hostname using the hostname command.

assuming this is happy in ci however lets keep it simple.

Reply:
ok this seam to fail agin 

SERVER: Checking cert hostname match                                         : WARN (Certificate /etc/pki/libvirt/servercert.pem owner does not match the hostname np0004809514.novalocal)

so it looks like it wants the fqdn not the hostname